framework/source/class/qx/core/BaseInit.js

```diff
@@ -40,7 +40,7 @@ qx.Class.define("qx.core.BaseInit",
      * @return {qx.core.Object} The application instance.
      */
     getApplication : function() {
-      return this.__application || null;
+      return qx.core.BaseInit.__application || null;
```

Comment:
does `__application` survive the name mangling ... for the build version ... or does this still not happen in the compiler and only in the old infrastructure?

Reply:
yes, although the compiler does not do private name mangling yet

Reply:
I think it would really help to get this name mangling in before we release as otherwise, we will never be able to put it in as it will break too much code of people who have ignored our conventions ...

Reply:
mangling is in now